src/Microsoft.ML.Mkl.Components/OlsLinearRegression.cs

```diff
@@ -33,7 +33,7 @@ namespace Microsoft.ML.Trainers
 {
     /// <summary>
     /// The <see cref="IEstimator{TTransformer}"/> for training a linear regression model using
-    /// ordinary least squares (OLS) for estimating the parameters of the linear regression model.
+    /// <a href="https://en.wikipedia.org/wiki/Ordinary_least_squares">ordinary least squares (OLS)</a> for estimating the parameters of the linear regression model.
```

Comment:
I remember some links being broken when there were parentheses. 
If it is confirmed to be an issue maybe we could omit (OLS) from the link.

Reply:
Should not be an issue for `<a>` inside xml.